packages/scaffold-ui/src/partials/w3m-account-default-widget/index.ts

```diff
@@ -220,7 +220,7 @@ export class W3mAccountDefaultWidget extends LitElement {
   }
 
   private isAllowedNetworkSwitch() {
-    const { requestedCaipNetworks } = NetworkController.state
+    const requestedCaipNetworks = NetworkController.getRequestedCaipNetworks()
```

Comment:
> [!INFO]
> Since we have a special logic while getting the requested CAIP networks. We are using this function rather than directly reaching to `.state.requestedCaipNetworks`